app/services/compliance-status.js

```diff
@@ -0,0 +1,102 @@
+import Ember from 'ember';
```

Comment:
@gib - This is the service I mentioned earlier.  It encapsulates all of the requests needed to render almost all of the compliance screens.  Rather than calling the `store` in each route, you can instead just inject this service into a route and return it in the `model` hook.  All of the compliance status checks are isolated  into the `CriterionStatus` classes that this service initiates.